fdbclient/vexillographer/fdb.options

```diff
@@ -109,6 +109,9 @@ description is not currently required but encouraged.
             description="Searches the specified path for dynamic libraries and adds them to the list of client libraries for use by the multi-version client API. Must be set before setting up the network." />
     <Option name="disable_local_client" code="64"
             description="Prevents connections through the local client, allowing only connections through externally loaded client libraries. Intended primarily for testing." />
```

Comment:
Although this option is implied and you don't need to call it in this feature, I think we can probably remove the "intended primary for testing" caveat. 
```suggestion
            description="Prevents connections through the local client, allowing only connections through externally loaded client libraries." />
```

Reply:
fixed.  thanks.